examples/vite-react-app/src/index.css

```
@@ -1,5 +1,5 @@
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-family: Figtree, system-ui, Avenir, Helvetica, Arial, sans-serif;
```

keep it as default here, no need to change it 
```suggestion
  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
```